cfgov/v1/management/commands/delete_page.py

```diff
@@ -0,0 +1,24 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from v1.models import CFGOVPage
+
+
+class Command(BaseCommand):
+    help = 'Deletes a page by its slug name'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--slug',
```

Comment:
Slugs [are not unique](https://github.com/wagtail/wagtail/blob/v1.7/wagtail/wagtailcore/models.py#L305) across the site, only within a particular directory. We ran into this with the D&R conference registration script, unfortunately you might have to specify page primary key instead.